### ISSUE TITLE
Missing FloatingPictures property

### ISSUE THREAD

**Issue by stsje** [OfficeDev/office-js#2832]
## Missing floatingPictures property

We seem to lack a way of retrieving floating pictures through the API.
Right now we're only able to retrieve inline pictures by using the `inlinePictures` property. It would be great to have the same for floating pictures :)
Currently there is no way of getting floating pictures at all.

Ideally the API should be something like
`context.document.body.floatingPictures`

Example of a floating picture in Word
![image](https://user-images.githubusercontent.com/15363306/189874297-c7c2fdb1-9d6c-458a-8a17-5f8045719bed.png)

**Comment by JinghuiMS** [comment]
@stsje  Thanks for reaching us. 
Can I know which platform(Word Desktop, Word Online, Mac or iOS) you are seeing this error ? And what's the word version ? it will be helpful for us to better understand your scenario.

**Comment by stsje** [comment]
@JinghuiMS yes of course!
All Word platforms, and any version of the API should be missing this feature. The preview API does not have this feature either https://docs.microsoft.com/en-us/javascript/api/word/word.body?view=word-js-preview

Tested with Microsoft 365 Word and Office-js 1.3

**Comment by jipyua** [comment]
@stsje you are right that so far only inline pictures are supported. We track Office Add-in feature requests on our [Microsoft 365 Developer Platform Ideas Forum]( https://aka.ms/m365dev-suggestions). Please add your request there. Feature requests there are considered when we go through our planning process. Thanks.

**Comment by stsje** [comment]
@jipyua I have opened a feature request as suggested https://techcommunity.microsoft.com/t5/microsoft-365-developer-platform/word-js-api-add-support-for-floating-pictures/idi-p/3633626

**Comment by stsje** [comment]
Is this feature being planned, or what is the status?

**Comment by jipyua** [comment]
hi @stsje thanks for checking again. In order for us to better understand and triage the priority, would you please provide more information about the business impact of missing these APIs to your add-in. And the typical user scenarios for these APIs in your business. Thanks.
